target/linux/ath79/dts/qca9533_tplink_cpe210-v3.dts

```diff
@@ -0,0 +1,139 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+	compatible = "tplink,cpe210-v3", "qca,qca9533";
+	model = "TP-LINK CPE210 v3";
+
+	extosc: ref {
```

Comment:
This is already defined in qca953x.dtsi, so this node can be dropped.

Reply:
done